### PR TITLE
[EdgeTPU] Add shell script for the prerequisites

### PR DESCRIPTION
**Pull request by profornnan**
Add the shell script that add PPA to get edgetpu-compiler package for EdgeTPU

ONE-vscode-DCO-1.0-Signed-off-by: profornnan <profornnan@naver.com>

Related: #1621 #1628

### CODE REVIEW

**Comment by jyoungyun** [comment]
Please add @Samsung/one-vscode as a reviewer. :)

File: script/prerequisitesForGetEdgeTPUToolchain.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# This shell script add PPA to get edgetpu-compiler package for EdgeTPUToolchain.
+
+curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
+echo "deb https://packages.cloud.google.com/apt coral-edgetpu-stable main" | sudo tee /etc/apt/sources.list.d/coral-edgetpu.list
+
+apt-get update -yqq
```
Comment (jyoungyun) [inline script/prerequisitesForGetEdgeTPUToolchain.sh:8]:
```suggestion
sudo apt-get update -yqq
```

This command needs `sudo` privileged. :)

Comment (profornnan) [reply script/prerequisitesForGetEdgeTPUToolchain.sh]: Thank you~!! I have added sudo privilege in the new commit :)

**Comment by dayo09** [comment]
Doesn't this script need executable permission? `prerequisitesForGetOneToolchain.sh` has executable permission.

```
-rw-rw-r--  1 dayo dayo  331  9월 22 13:22 prerequisitesForGetEdgeTPUToolchain.sh
-rwxrwxr-x  1 dayo dayo  184  9월 14 16:55 prerequisitesForGetToolchains.sh
-rwxrwxr-x  1 dayo dayo  933  9월  5 19:18 sdbSpecList.sh
```

**Review by dayo09 — APPROVED** [review]
LGTM, Thanks!

**Comment by profornnan** [comment]
Thank you!! :D

**Review by jyoungyun — APPROVED** [review]
LGTM

**Pull request merged**